blocks-common/b-page/__csp/b-page__csp.bemhtml

```diff
@@ -0,0 +1,41 @@
+block b-page, elem csp {
+
+    bem: false
+
+    tag: 'meta'
+
+    attrs: {
+        'http-equiv': 'Content-Security-Policy',
+        'content': Object.keys(this.ctx.policies).map(function(name) {
+            return name + ' ' + this.ctx.policies[name].join(' ') + ';';
+        }, this).join(' ')
+    }
+
+    default: {
+        var ctx = this.ctx,
+            nonce = this._nonce,
+            defaultPolicies = {
+                'default-src': [ "'self'"]
+            };
+
+        ctx.policies = this.extend(defaultPolicies, ctx.policies);
+
+        // Параметр nonce распространяется только на <script> и <style>.
+        // http://www.w3.org/TR/CSP/#script-src-the-nonce-attribute
+        if(nonce) {
+            if(ctx.policies['script-src']) {
+                ctx.policies['script-src'].push("'nonce-" + nonce + "'");
+            } else {
+                ctx.policies['script-src'] = ["'nonce-" + nonce + "'"];
+            }
+
+            if(ctx.policies['style-src']) {
+                ctx.policies['style-src'].push("'nonce-" + nonce + "'");
+            } else {
+                ctx.policies['style-src'] = ["'nonce-" + nonce + "'"]
+            }
+        }
+
+        applyNext();
```

Comment:
not necessary right now but would be good to use explicit `return` for apply* (for compatibility with `bem-xjst@2.x`)